website/docs/advanced/verification-methods.md

```diff
@@ -4,16 +4,25 @@ title: Extending Verification Methods
 sidebar_label: Verification Methods
 ---
 
-As explained by our [Verifier ADR](https://github.com/Open-Attestation/adr/blob/master/verifier.md), the library is composed of verification methods, that are run over a document and provide fragments containing details about each verification. In this guide we will learn how to write custom verification method and how you can distribute your own verifier.
+As explained by our [Verifier ADR](https://github.com/Open-Attestation/adr/blob/master/verifier.md), the library is composed of verification methods that are run over a document and provide fragments containing details about each verification. In this guide, we will use the contents in a _Version 2_ document as an example and learn how to write custom verification methods and distribute your own custom verifier.
 
 ### Building a custom verification method
+
 We will write a verification method having the following rules:
+
 1. it must run only on document having their version equal to `https://schema.openattestation.com/2.0/schema.json`.
 1. it must return a valid fragment, if and only if the document data hold a name property with the value `Certificate of Completion`
 
 ### Document version must be equal to `https://schema.openattestation.com/2.0/schema.json`
-This is where `skip` and `test` methods comes into play. We will use the `test` method to return when the verification method run, and the `skip` method to explain why it didn't run:
+
+This is where `skip` and `test` methods come into play. We will use the `test` method to return when the verification method run, and the `skip` method to explain why it didn't run:
+
 ```javascript
```

Comment:
can you open an issue to migrate our code to ESM / TS

Reply:
ok done. https://github.com/Open-Attestation/documentation-website/issues/75